### ISSUE TITLE
Missing documentation for conditions

### ISSUE THREAD

**Issue by colesmj** [OWASP/pytm#28]
Some properties/conditions are obvious and don't really need documentation, such as protocol or isEncrypted. But other properties may not be obvious to some, such as Dataflow.authenticatedWith - with what?

**Comment by colesmj** [comment]
@shambho Does your commit 3 days ago resolve this issue? If so, let's close this one.
Alternately, or maybe also, we should document the object model on the wiki. I had put one up there a long time ago, but it is no longer valid, so I will deprecate it (I'd like to keep it around for some future looking work).

**Comment by shambho** [comment]
@colesmj nope. Commit from 3 days ago was about adding some cosmetic changes to tm.py, Readme and pytm.py. Agree, we should have more documentation on the properties available in each Class.

**Comment by nineinchnick** [comment]
#91 documents a very small subset of attributes.

**Comment by cptfixit** [comment]
I would like to second this issue report: flags such as "sourceFiles" are not clear to the average user/reader, without explanation.

**Comment by Dakes** [comment]
+1 Being new to threat modeling, some of these are quite hard to understand. Some can be figured out with a google search, like `usesStrongSessionIdentifiers`.
But for `checksInputBounds`, I had to check in the threats.json to understand, that it refers to Buffer sizes. So some additional documentation for many properties would certainly be beneficial.
